### ISSUE TITLE
Catched errors from serviceCall.requestResponse are of type 'string' instead of Error

### ISSUE THREAD

**Issue by vvpyshnenko** [scalecube/scalecube-js#186]
## Describe the bug

It is expected the errors thrown out serviceCall.requestResponse call should be instances of regular js Error. However it turns out of type 'string'

## Reproduce
```typescript
  serviceCall
    .requestResponse(data)
    .catch((e) => {
      console.log('typeof error is ', typeof e); // => typeof error is string
    });
```

#### Expected behavior

A test case in _**Gherkin Syntax**_ or the link to this test case.

#### Actual behavior

How the test case failed.

#### Code

Sandbox, snippet, repository etc..

#### Frequency

100% reproduce / happens occasionally / hard to reproduce

#### Attachments

Screenshots, videos, documents, localStorage and cookies data etc..

#### Environment

- OS & Version: [e.g. iOS:11.10...]
- Browser & Version [e.g. chrome:x..., safari:y..]
- Device (the full name of the device)

**Comment by katzIdo** [comment]
fixed #203

**Comment by katzIdo** [comment]
fixed #203
